### PR TITLE
Collections documentation fixes

### PR DESCRIPTION
**Pull request by qurbonzoda**

### CODE REVIEW

File: pages/docs/reference/collection-write.md [hunk 1]
```diff
@@ -28,14 +28,7 @@ fun main() {
 ```
 </div>
 
-[`addAll()`](/api/latest/jvm/stdlib/kotlin.collections/add-all.html) adds every element of the argument object to a list or a set.
```
Comment (qurbonzoda) [inline pages/docs/reference/collection-write.md]: I believe these two removed paragraphs are rephrased below.

File: pages/docs/reference/collection-ordering.md [hunk 2]
```diff
@@ -208,7 +208,7 @@ However, if the mutability of the list is unknown or the source is not a list at
 
 ## Random order
 
-Finally, there is a shuffle function that retrieves all collection elements in a random order - [`shuffled()`](/api/latest/jvm/stdlib/kotlin.collections/shuffled.html).
+Finally, there is a shuffle function that returns a new `List` containing the collection elements in a random order - [`shuffled()`](/api/latest/jvm/stdlib/kotlin.collections/shuffled.html).
```
Comment (qurbonzoda) [inline pages/docs/reference/collection-ordering.md]:
```suggestion
Finally, there is a function that returns a new `List` containing the collection elements in a random order - [`shuffled()`](/api/latest/jvm/stdlib/kotlin.collections/shuffled.html).
```

File: pages/docs/reference/collection-transformations.md [hunk 3]
```diff
@@ -106,7 +106,7 @@ fun main() {
 ```
 </div>
 
-When you have a `List` of `Pair`s, you can do the reverse transformation – _unzipping_ – that builds two lists from these `Pairs`:
+When you have a `List` of `Pair`s, you can do the reverse transformation – _unzipping_ – that builds two lists from these `Pair`s:
```
Comment (ilya-g) [inline pages/docs/reference/collection-transformations.md]:
```suggestion
When you have a `List` of `Pair`s, you can do the reverse transformation – _unzipping_ – that builds two lists from these pairs:
```

File: pages/docs/reference/collection-write.md (hunk 1 above)
Comment (ilya-g) [reply pages/docs/reference/collection-write.md]: @p7nov please help choosing which one to leave.

File: pages/docs/reference/map-operations.md [hunk 4]
```diff
@@ -168,7 +168,7 @@ fun main() {
 ```
 </div>
 
-To add multiple entries at a time, use [`putAll()`](/api/latest/jvm/stdlib/kotlin.collections/put-all.html). Its argument can be a `Map` or a group of `Pairs`: `Iterable`, `Sequence`, or `Array`.
+To add multiple entries at a time, use [`putAll()`](/api/latest/jvm/stdlib/kotlin.collections/put-all.html). Its argument can be a `Map` or a group of `Pair`s: `Iterable`, `Sequence`, or `Array`.
```
Comment (ilya-g) [inline pages/docs/reference/map-operations.md]: same here: pairs

File: pages/docs/reference/collection-write.md (hunk 1 above)
Comment (p7nov) [reply pages/docs/reference/collection-write.md]: Yep, this is correct.  Seems like I forgot to remove the old description when rewriting it.

**Pull request merged**